src/content/getting-started/developers/vanilla.mdx

```diff
@@ -89,6 +89,8 @@ We have individual [CodePens](https://codepen.io/collection/XqRbEz/) for all of
 Using the Carbon Sass files infers usage of the SCSS pre-processor.
 All Sass files use the `*.scss` file extension.
 
+For transpiling Sass code, use `node-sass` based Sass compilers, for example, WebPack `sass-loader` and `gulp-sass`.
```

Comment:
should this be 'or' rather than 'and'?
```For transpiling Sass code, use `node-sass` based Sass compilers, for example, WebPack `sass-loader` or `gulp-sass`.```

Reply:
Good point @chrisconnors-ibm - Updated as such: https://github.com/carbon-design-system/carbon-website/pull/1669/commits/6397253ff43a45aa2fb599ce547b50567faa8f53